app/_hub/kong-inc/ai-azure-content-safety/overview/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+nav_title: Overview
+---
+
+The Azure Content Safety plugin allows administrators to enforce 
+introspection with the [Azure Content Safety](https://azure.microsoft.com/en-us/products/ai-services/ai-content-safety) service 
+for all requests handled by the AI Proxy plugin.
+The plugin enables configurable thresholds for the different moderation categories 
+and an array set of pre-configured blocklist IDs can be specified from your Azure Content Safety instance.
+
+Audit failures can be observed and reported on using the {{Site.base_gateway}} logging plugins.
+
+{:.note}
+> This plugin extends the functionality of the [AI Proxy plugin](/hub/kong-inc/ai-proxy/), and requires AI Proxy to be configured first. 
+Check out the [AI Gateway quickstart](/gateway/latest/get-started/ai-gateway/) to get an AI proxy up and running within minutes!
+
+## Format
+
+This plugin works with all of the AI Proxy plugin's `route_type` settings (excluding the `preserve` mode), and is able to
+compose an Azure Content Safety text check by compiling all chat history, or just the `'user'` content.
+
+## Get started with the AI Prompt Decorator plugin
+
+* [AI Gateway quickstart: Set up AI Proxy](/gateway/latest/get-started/ai-gateway/)
+* [Configuration reference](/hub/kong-inc/ai-azure-content-safety/configuration/)
+* [Basic configuration example](/hub/kong-inc/ai-prompt-decorator/how-to/basic-example/)
+* [Learn how to use the plugin](/hub/kong-inc/ai-prompt-decorator/how-to/)
+
+### Other AI plugins
+
+You may also be interested in the following AI plugins:
+* [AI Proxy](/hub/kong-inc/ai-proxy/)
+* [AI Request Transformer](/hub/kong-inc/ai-request-transformer/)
+* [AI Response Transformer](/hub/kong-inc/ai-request-transformer/)
```

Comment:
```suggestion
* [AI Response Transformer](/hub/kong-inc/ai-request-transformer/)
{% if_version gte:3.7.x %} 
* [AI Rate Limiting Advanced](/hub/kong-inc/ai-rate-limiting-advanced/)
{% endif_version %}
```
This one was added to AI Proxy's homepage, I wonder if this Other AI plugins section should be an include since it's used on all the AI plugin overview pages. It would be easier to maintain that way.

Reply:
Hmm. We could just rename it to "All AI plugins" or just "AI plugins" and put them in an include, like you said. Right now the list is different for each plugin since it doesn't contain a link to the plugin you're already on. But I don't think it's a big deal to do that.

Reply:
Going to do this separately, outside of this plugin's PR. This PR is already getting overloaded with extra/not the most related content.